.github/workflows/tests.yml

```diff
@@ -175,6 +176,15 @@ jobs:
             ;;
           esac
 
+      - name: Install TeX Live dependencies
+        run: |
+          case "${{ runner.os }}" in
+          Linux)
+            mkdir -p $HOME/texmf
+            tlmgr init-usertree
+            tlmgr install --force dejavu heuristica
+          esac
```

Comment:
```suggestion
        if: startsWith(runner.os, 'Linux')
        run: |
          mkdir -p $HOME/texmf
          tlmgr init-usertree
          tlmgr install --force dejavu heuristica
```

Reply:
Yeah, it bails out with

    tlmgr: Local TeX Live (2021) is older than remote repository (2025).